src/alloc-aligned.c

```diff
@@ -61,53 +61,53 @@ static void* mi_heap_malloc_zero_aligned_at(mi_heap_t* const heap, const size_t
 }
 
 
-void* mi_heap_malloc_aligned_at(mi_heap_t* heap, size_t size, size_t alignment, size_t offset) mi_attr_noexcept {
+mi_decl_allocator void* mi_heap_malloc_aligned_at(mi_heap_t* heap, size_t size, size_t alignment, size_t offset) mi_attr_noexcept {
```

Comment:
:-( brrr, a bit ugly but probably worth it to reduce compiler warnings, sigh.

Reply:
The alternative would be to remove those attributes from the declarations... I'm not sure whether that would be appropriate, though, I am not familiar enough with the code base.

Reply:
Ah we need `mi_decl_allocator` for MSVC to generate better code (as alloc'd pointers are then considered non-aliased)